backend/lucene/src/main/java/org/hibernate/search/backend/lucene/logging/impl/Log.java

```diff
@@ -68,10 +76,21 @@
 			value = "Unable to close the index reader. %1$s")
 	void unableToCloseIndexReader(@FormatWith(EventContextFormatter.class) EventContext context, @Cause Exception e);
 
+	@LogMessage(level = WARN)
+	@Message(id = ID_OFFSET_1 + 75,
+			value = "Configuration setting " + SearchBackendLuceneSettings.LUCENE_VERSION
+			+ " was not specified: using LATEST. %1$s")
+	void recommendConfiguringLuceneVersion(@FormatWith(EventContextFormatter.class) EventContext context);
```

Comment:
I wonder if we should provide a sensible value depending on the current Lucene version. That might help people initializing an application.

Reply:
Not sure that' what you meant, but I added the exact version to the log message (7.y.z).